File: src/app/models/Field.ts
export class Field{
  id: number;
  map: string;
  where: number;
  userId: number;
  buildingId: number;
  buildingLevel: number;

  constructor(id: number, map: string, where: number, userId: number, buildingId: number, buildingLevel: number) {
    this.id = id;
    this.map = map;
    this.where = where;
    this.userId = userId;
    this.buildingId = buildingId;
    this.buildingLevel = buildingLevel;
  }
}

File: src/main/java/com/kam/browser/model/Field.java
package com.kam.browser.model;

import lombok.*;


import javax.persistence.*;

@Entity
@Data
@NoArgsConstructor
@AllArgsConstructor
@Builder
public class Field {

  @Id
  @GeneratedValue
  private Long id;
  @ManyToOne(fetch = FetchType.LAZY)
  @Getter(value=AccessLevel.NONE)
  private User user;
  private String map;
  private Integer fieldNumber;
  private Integer cols;
  private Integer rows;
  private String className;
  private Integer buildingId;
  private Integer buildingLevel;
}
